.github/pull_request_template.md

```diff
@@ -1,55 +1,34 @@
-<!-- You are awesome! Your contribution to RocketPy is fundamental in our endeavour to create the next generation solution for rocketry trajectory simulation! -->
-
-<!-- You may use this template to describe your Pull Request. But if you believe there is a better way to express yourself, don't hesitate! -->
-
```

Comment:
Personally, I believe these lines should be kept. They are just comments, but are important to welcome contributions and emphasize that the template is not mandatory.